pycode/memilio-epidata/memilio/epidata_test/test_epidata_geoModificationGermany.py

```diff
@@ -104,15 +113,14 @@ class Test_geoModificationGermany(fake_filesystem_unittest.TestCase):
     countytoregion_zfill_string = {
         "'01001': '11'", "'11000': '00'", "'05362': '16'", "'03452': '02'",
         "'01054': '11'", "'16077': '17'"}
-    regiontocounty_string = {
+    regiontocounty_string = [
```

Comment:
Why do you change regiontocouty_string for set into list but not countytoregion_string?

Reply:
Now, I generally replaced sets by lists.